.gitlab/binary_build/serverless.yml

```diff
@@ -0,0 +1,27 @@
+---
+.if_version_7: &if_version_7
+  if: $RELEASE_VERSION_7 != ""
+
+.retrieve_linux_go_deps: &retrieve_linux_go_deps |
+  mkdir vendor && tar xzf vendor.tar.gz -C vendor
+  rm -f vendor.tar.gz
+
+build_serverless-deb_x64:
+  stage: binary_build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:main", "size:large"]
+  needs: ["linux_x64_go_deps"]
+  before_script:
+    - *retrieve_linux_go_deps
+  script:
+    - cd cmd/serverless && go build -a -v -tags serverless
```

Comment:
I would move this to an invoke task: it's what we do for all other agent flavors and it will help us keep all `build_serverless` jobs

Reply:
I agree, I'll do that in a next commit.